src/main/java/seedu/address/model/person/Loan.java

```diff
@@ -22,40 +22,45 @@ public class Loan {
     private final Date startDate;
     private final Date returnDate;
     private boolean isReturned;
+    private Person assignee;
 
     /**
      * Constructs a {@code Loan} with a given id.
      *
-     * @param id A valid id.
-     * @param value A valid value.
-     * @param startDate A valid start date.
+     * @param id         A valid id.
+     * @param value      A valid value.
+     * @param startDate  A valid start date.
      * @param returnDate A valid return date.
+     * @param assignee   A valid assignee.
      */
-    public Loan(int id, float value, Date startDate, Date returnDate) {
-        requireAllNonNull(id, value, startDate, returnDate);
+    public Loan(int id, float value, Date startDate, Date returnDate, Person assignee) {
+        requireAllNonNull(id, value, startDate, returnDate, assignee);
         this.id = id;
         this.value = value;
         this.startDate = startDate;
         this.returnDate = returnDate;
         this.isReturned = false;
+        this.assignee = assignee;
     }
 
     /**
      * Constructs a {@code Loan} with a given id and return status.
      *
-     * @param id A valid id.
-     * @param value A valid value.
-     * @param startDate A valid start date.
+     * @param id         A valid id.
+     * @param value      A valid value.
+     * @param startDate  A valid start date.
      * @param returnDate A valid return date.
      * @param isReturned A valid return status.
+     * @param assignee   A valid assignee.
      */
-    public Loan(int id, float value, Date startDate, Date returnDate, boolean isReturned) {
-        requireAllNonNull(id, value, startDate, returnDate, isReturned);
+    public Loan(int id, float value, Date startDate, Date returnDate, boolean isReturned, Person assignee) {
```

Comment:
Do we need a constructor that indicates a `isReturned` status?

Reply:
It may be useful for reading from the storage file.